### PR TITLE
add data_science_2 exercise

### PR DESCRIPTION
**Pull request by miguelsimon**
Another basic data science exercise:

* learn to compare binary classifiers
* calculate confusion matrices
* sneak in some decision theory by comparing losses of different decisions

### CODE REVIEW

File: data_science_2/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Vistos los datos de accuracy en el "viaje de trabajo" a Hawaii financiado por GSKPAZ, el Ministerio de Sanidad está a punto de aprobar una campaña de screening masivo usando el test `ultradetect` pero a un funcionario de bajo nivel [le huele a chamusquina](https://en.wikipedia.org/wiki/Bad_Pharma) y te ha encargado que verifiques los cálculos y las conclusiones.
+
+Los datos de los trials vienen en csv, con las siguientes columnas:
+
+* `test_positive`: si el test de detección precoz ha dado un resultado positivo **1** o negativo **0**
+* `sick`: si el seguimiento clínico del paciente ha determinado que estaba enfermo de FIC **1** o no enfermo **0**
+
+#### ejercicio 0
+
+Como los trials han usado muestras poblacionales aleatorias podemos estimar [la prevalencia](https://es.wikipedia.org/wiki/Prevalencia) del FIC haciendo pooling de los datos de ambos trials.
```
Comment (valentinboyanov) [inline data_science_2/README.md]: I've noticed that this is the only reference link in Spanish. Is it intentional?

I don't understand what "data pooling" means in this context. Does it mean combining the data from both trials?

**Review by valentinboyanov — COMMENTED** [review]
I've added some doubts, but I think I have been able to understand the exercise in general, the links work and it seems interesting and challenging!

File: data_science_2/README.md (hunk 1 above)
Comment (miguelsimon) [reply data_science_2/README.md]: Nope, wasn't intentional, fixed.

I've clarified what's to be pooled & added link to wiki on pooling epidemiological data.

**Review by valentinboyanov — APPROVED** [review]
👍

**Pull request merged**